test/e2e/puppeteer.js

```diff
@@ -28,17 +57,52 @@ const exceptionList = [
 	'webgl_worker_offscreencanvas', // in a worker, not robust
 
 	// TODO: most of these can be fixed just by increasing idleTime and parseTime
```

Comment:
Seems like the Chromium on Mac is very unstable... I will cleanup this list in a further PR (will try to determine which examples can be fixed just by increasing `idleTime` and `parseTime`, which examples require screenshot regeneration, and which examples simply break on different OSes e.g. Mac).

Reply:
I had only one test fail on Mac.  Ran it again using the 'run' command it provided in the message, it completed instantly and successfully.  

I'm not sure of the success was just image generation or both "image generation and comparison".

It added the image to the source of the project so I needed to discard the changed file.  Is it using hardware acceleration or skipping the compare when you run the command it provides for a failed test ?  It's very fast.

Reply:
The problem is that with each run some small number of random examples fail (about 1-7). On Windows and Ubuntu such thing doesn't happen.

Reply:
I have a few that need retries on Windows.  They don't fail.  Some of them are just yellow, maybe they are supposed to be yellow to test failure conditions.

Reply:
With the current exceptions listed almost each Windows and Linux run succeeds. And almost any Mac run still faila.

Reply:
I'm running another now on mac.  I've had mostly successes except the last one.  Can I offer logs or dumps that may help ?

Reply:
It just downloaded chromium again on the Mac and on Windows.  I did a pull an hour ago.  That normal ? If we are using the daily chromium builds, things might break and unbreak randomly ?

Reply:
The method is downloading is taken from https://www.chromium.org/getting-involved/download-chromium/. I assume that is the Canary builds.
From https://www.chromium.org/getting-involved/dev-channel/:
> The release channels for chrome range from the most stable and tested (Stable channel) to completely untested and likely least stable (Canary channel). You can run all channels alongside all others, as they do not share profiles with one another. This allows you to play with our latest code, while still keeping a tested version of Chrome around.

> * Stable channel: This channel has gotten the full testing and blessing of the Chrome test team, and is the best bet to avoid crashes and other issues. It's updated roughly every two-three weeks for minor releases, and every 6 weeks for major releases.
> * Beta channel: If you are interested in seeing what's next, with minimal risk, Beta channel is the place to be. It's updated every week roughly, with major updates coming every six weeks, more than a month before the Stable channel will get them.
> * Dev channel: If you want to see what's happening quickly, then you want the Dev channel. The Dev channel gets updated once or twice weekly, and it shows what we're working on right now. There's no lag between major versions, whatever code we've got, you will get. While this build does get tested, it is still subject to bugs, as we want people to see what's new as soon as possible.
> * **Canary build: Canary builds are the bleeding edge. Released daily, this build has not been tested or used, it's released as soon as it's built.**
> * Other builds: If you're extra brave, you can download the latest working (and that's a very loose definition of working) build from [download-chromium.appspot.com](https://download-chromium.appspot.com/). You can also look for a more specific recent build by going to [the Chromium continuous build waterfall](http://build.chromium.org/), looking at the number near the top under "LKGR", and then going to [this Google Storage bucket](http://commondatastorage.googleapis.com/chromium-browser-continuous/index.html) and downloading the corresponding build.
Note: Early access releases (Canary builds and Dev and Beta channels) will be only partly translated into languages other than English. Text related to new features may not get translated into all languages until the feature is released in the Stable channel.

I couldn't find a way to download e.g. the latest build from the dev channel. The commit numbers taken from OmahaProxy very often don't work.

Reply:
The same Mac test fails each time.  Running again to update this comment with the name.

Reply:
> The same Mac test fails each time.

Yes, and each time it fails with different examples. I'm guessing it's related to https://github.com/puppeteer/puppeteer/issues/8693 -- testing it.

Reply:
> I couldn't find a way to download e.g. the latest build from the dev channel. The commit numbers taken from OmahaProxy very often don't work.

Maybe just easier to continue updating the version by hand from time to time?

Reply:
I'll try to make a PR that uses OmahaProxy to download.